src/chi/components/tabs/tabs.scss

```diff
@@ -302,6 +302,24 @@
       }
     }
   }
+
+  &.-toolbar {
```

Comment:
@davidojedacl shouldn't this be done in header scss as this is an specific case of header toolbar? @mattnickles thoughts